src/radical/pilot/configs/resource_nioz.json

```diff
@@ -0,0 +1,40 @@
+
+{
+    "laplace":
+    {
+        "description"                 : "",
```

Comment:
@andre-merzky just to be consistent with other resource config files, can we add a description or at least the official website `URL` of this cluster/HPC? It would be easier for us. (for documentation purposes etc.) Other than that it looks fine to me. Thanks.

Reply:
will do!